src/components/Navbar/index.tsx

```diff
@@ -0,0 +1,167 @@
+import { styled, useTheme } from '@mui/material/styles';
+import Box from '@mui/material/Box';
```

Comment:
everything coming from '@mui/material' should all be in a single import statement e.g import {Box, Typography} from '@mui/material'